python/tests/test_rlp.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from trezorlib import _rlp
+
+VECTORS = (  # data, expected
+    (b"\x10", b"\x10"),
+    (b"dog", b"\x83dog"),
+    (b"A" * 1024, b"\xb9\x04\x00" + b"A" * 1024),
+    ([b"dog", b"cat", [b"spy"]], b"\xcd\x83dog\x83cat\xc4\x83spy"),
+    ([b"A" * 1024], b"\xf9\x04\x03\xb9\x04\x00" + b"A" * 1024),
+    ([], b"\xc0"),
+    (b"", b"\x80"),
+    (1, b"\x01"),
+    (0x7F, b"\x7f"),
+    (0x80, b"\x81\x80"),
+    (0x1_0000_0001, b"\x85\x01\x00\x00\x00\x01"),
+    ([0x1234, 0x5678], b"\xc6\x82\x12\x34\x82\x56\x78"),
```

Comment:
nit: testcase for the `len > 55` branch would be nice

Reply:
eba3577832a788ab7cc56008551410f88b71224b